tools/prepare-source.sh

```diff
@@ -2,20 +2,20 @@
 set -eux
 
 GO_VERSION=1.22.0
+OAPI_VERSION=2.4.1
+
 GO_BINARY=$(go env GOPATH)/bin/go$GO_VERSION
-OAPI_VERSION=2.3.0
+TOOLS_PATH="$(realpath "$(dirname "$0")/../bin")"
```

Comment:
hmm, should we move this under `tools/bin`, that way it's clear these are just dependencies for tools? `/bin` could be or become the folder that we actually build into.